cypress/features/build/build.feature

```diff
@@ -22,73 +22,162 @@ Feature: Build tests
       And I open component preview no iframe
     Then "<data-component>" component is visible
     Examples:
-      | component           | data-component     |
-      | dialog-full-screen  | dialog-full-screen |
-      | dialog              | dialog             |
-      | flash               | flash              |
-      | pages               | page               |
-      | confirm             | confirm            |
+      | component          | data-component     |
+      | dialog-full-screen | dialog-full-screen |
+      | dialog             | dialog             |
+      | flash              | flash              |
+      | pages              | page               |
+      | confirm            | confirm            |
 
   @build
   Scenario Outline: Component <component> without activation button
     When I open "<component>" component iframe
```

Comment:
Please add 'in' before iframe. We should try to keep the convention using similar methods, e.g below you used `When I open "<component>" component page validations **in iframe**`
```suggestion
    When I open "<component>" component in iframe
```

Reply:
fixed